scripts/common/object_store.sh

```diff
@@ -66,6 +66,59 @@ function object_store_bucket_exists() {
         "http://$addr/$bucket" >/dev/null 2>&1
 }
 
+# migrate_object_store creates a pod that migrates data between two different object stores. receives
+# the namespace, the source and destination addresses, access keys and secret keys. returns once the
+# pos has been finished or a timeout of 5 minutes has been reached.
+function migrate_object_store() {
+    local namespace=$1
+    local source_addr=$2
+    local source_access_key=$3
+    local source_secret_key=$4
+    local destination_addr=$5
+    local destination_access_key=$6
+    local destination_secret_key=$7
+
+    kubectl delete pod sync-object-store ${namespace} --force --grace-period=0 &>/dev/null || true
```

Comment:
Is the intention here to ignore not found? if so perhaps use the flag. also you are missing `-n`

```suggestion
    kubectl -n "$namespace" delete pod sync-object-store --force --grace-period=0 --ignore-not-found
```